ui/pages/confirmations/components/confirm/info/token-transfer/token-transfer.tsx

```diff
@@ -1,5 +1,8 @@
+import React from 'react';
+import SendHeading from '../shared/send-heading/send-heading';
+
 const TokenTransferInfo = () => {
-  return null;
+  return <SendHeading />;
```

Comment:
Are we going to extend this component later or will it stay just importing this?

Reply:
Yes, this work continues in the next PR https://github.com/MetaMask/metamask-extension/pull/27654